CRM/Utils/Check/Component/Env.php

```diff
@@ -258,49 +258,63 @@ public function checkDefaultMailbox() {
   }
 
   /**
-   * Checks if cron has run in a reasonable amount of time
+   * Checks if cron has run in the past hour (3600 seconds)
    * @return array
+   * @throws CRM_Core_Exception
    */
   public function checkLastCron() {
     $messages = [];
 
     $statusPreference = new CRM_Core_DAO_StatusPreference();
     $statusPreference->domain_id = CRM_Core_Config::domainID();
-    $statusPreference->name = 'checkLastCron';
+    $statusPreference->name = __FUNCTION__;
```

Comment:
Just curious why this change? The JobManager still refers to it as 'checkLastCron', so while it's unlikely somebody will change this function name, that would then get out of sync. Should it be a class constant which could then be used in both places?

Reply:
@demeritcowboy the are actually the same thing, as the function name is "checkLastCron".

I had previously gotten confused about the relationship between the statusPreference and the function, and so I made this change to un-confuse myself. It's more consistent this way because the statusMessage generated below uses `__FUNCTION__` and not the string literal.